cernopendata/modules/records/serializers/schemaorg.py

```diff
@@ -23,14 +23,31 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 """CERN Opendata schema.org JSON-LD serializer."""
 
+import json
 
 from invenio_records_rest.serializers.json import JSONSerializer
 
 from cernopendata.modules.records.serializers.schemas \
     import schemaorg_schemas as schemas
 
 
-class CODSchemaorgSerializer(JSONSerializer):
+class BasicJSONSerializer(JSONSerializer):
+    """Basic JSON serializer."""
+
+    def serialize(self, pid, record, links_factory=None, **kwargs):
+        """Serialize a single record and persistent identifier.
+
+        :param pid: Persistent identifier instance.
+        :param record: Record instance.
+        :param links_factory: Factory function for record links.
+        """
+        return json.dumps(
+            self.transform_record(pid, record, links_factory, **kwargs),
+            indent=2,
+            separators=(", ", ": "),
+        )
```

Comment:
Do we even need this class? It's basically replicating [this functionality](https://github.com/inveniosoftware/invenio-records-rest/blob/c4a3717afcf9b08b6e42f3529addecc64bb2e47c/invenio_records_rest/serializers/json.py#L29-L32). If one passes `?prettyprint=1` the schema would get indented automatically. I guess for Google robots it doesn't matter that is indented or not and we can always force it. WDYT? cc/ @tiborsimko

Reply:
It may be useful for "data curators" to see the JSON nicely printed, with new lines and everything, when doing "view page source" in the browsers.  That was the only motivation.

That said, we could enrich our unit test case and check whether we produce wanted output there. This would alleviate need fro pretty formatting. We currently under-use the cernopendata test suite, and in order to catch problems like these early, it would be easily to add some seriarisation test cases, since we have all the fixtures present in the source code for CI to see as well.  

Anyway, pretty-printing JSON by default in our HTML output of detailed record pages may be useful for anyone accessing only the web site still, so I guess let's keep it?